app/views/consent_forms/review_match.html.erb

```diff
@@ -33,7 +33,7 @@
           timestamp: @consent_form.recorded_at
         },
         refusal_reason: {
-          reason: @consent_form.reason,
+          reason: @consent_form.human_enum_name(:reason).presence,
```

Comment:
This surprised me a little. I wonder if we should change `human_enum_name` to return `nil` instead of an empty string when the enum is `nil`. 👟🥫